src/pages/_components/landing-page/EcosystemPanel.astro

```diff
@@ -0,0 +1,63 @@
+---
+import type { ImageMetadata } from 'astro';
+import { Icon } from 'astro-icon/components';
+import Image from 'astro/components/Image.astro';
+import type { ComponentProps } from 'astro/types';
+import ThemeContainer from './ecosystem-tabs/ThemeContainer.astro';
+
+interface Props {
+	themes: ComponentProps<typeof ThemeContainer>['theme'][];
+	moreCard?: {
+		href: string;
+		image: ImageMetadata;
+		title: string;
+	};
+	moreLink?: {
+		href: string;
+		title: string;
+	};
+}
+
+const { moreCard, moreLink, themes } = Astro.props;
+---
+
+<div class="p-1 grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-4 rounded-2xl">
+	{themes.map((theme) => <ThemeContainer theme={theme} />)}
+	{
+		moreCard && (
+			<div class="size-fit group relative">
+				<Image
+					width={800}
+					class="width-full rounded-xl isolate select-none"
+					src={moreCard.image}
+					alt="An image showing all types of content driven websites you can build with Astro"
```

Comment:
What do you think about removing this?
```suggestion
					alt=""
```
I spotted it the other day and I don’t _think_ it’s at all helpful? The image seems to be decorative to me.

Reply:
This was part of the "other potential improvements in this area of the landing page" I mentioned so happy to go with it, thanks for suggesting it 🙌